mod/consensus/pkg/cometbft/service/params/consensus.go

```diff
@@ -47,6 +47,7 @@ func NewConsensusParamsStore(cs ChainSpec) *ConsensusParamsStore {
 // Get retrieves the consensus parameters from the store.
 // It returns the consensus parameters and an error, if any.
 func (s *ConsensusParamsStore) Get() *cmtproto.ConsensusParams {
```

Comment:
@fridrik01 I believe this is fine. I would suggest the following approach to drop the linting:
```
	cfg, _ := s.cs.GetCometBFTConfigForSlot(0).(*cmttypes.ConsensusParams)
	p := cfg.ToProto()
	return &p
```